UnitTests/resources/TestSolution/UnitTests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<packages>
+    <package id="NUnit" version="2.6.2" targetFramework="net472" />

```

Comment:
Could you add the version number to the name of `nunit.framework.dll` since it seems kind of important and essential